packages/plugin/src/api/analytics.js

```diff
@@ -38,28 +46,187 @@ export const apiFetchAnalyticsForYearOverYear = async (
         yearlySeriesReq
     )
 
-    const requests = []
+    const periodDates = []
     const yearlySeriesLabels = []
 
     const now = new Date()
     const currentDay = ('' + now.getDate()).padStart(2, 0)
     const currentMonth = ('' + (now.getMonth() + 1)).padStart(2, 0)
 
-    yearlySeriesRes.metaData.dimensions[peId].forEach(period => {
-        yearlySeriesLabels.push(yearlySeriesRes.metaData.items[period].name)
+    const peItems = layoutGetDimensionItems(visualization, peId)
+    console.log('pe items', peItems)
+
+    // relative week period in use
+    if (getRelativePeriodTypeUsed(peItems) === WEEKS) {
+        console.log('relative weeks period in use')
```

Comment:
leftover log from dev?

Reply:
Yes, along many others.
I cleaned up all the debug logs.